K8SPS-592: Document backup sourcePod (#170)
percona/k8sps-docs: docs/backups-ondemand.md

      storageName: s3-us-west
    EOF
    ```

## Specifying the backup source

> When you create a backup object, operator selects a pod to take the backup from. You can see the backup source pod in the backup object's status:

Reviewer: ```suggestion
When you create a backup object, the Operator selects a Pod to take the backup from. You can see the backup source pod in the backup object's status:
```

```
$ kubectl get ps-backup backup1 -o yaml
apiVersion: ps.percona.com/v1
kind: PerconaServerMySQLBackup
metadata:
  name: backup1
  ...
status:
  backupSource: cluster1-mysql-1.cluster1-mysql.<namespace>
  ...
```

> The source pod can also be specified in the backup object to run the backup on a specific pod:

Reviewer: ```suggestion
You can specify the source pod in the backup object to run the backup on this specific pod:
```
